src/pages/projectSettings/tabs/EventLogTab/EventLogTab.tsx

```diff
@@ -38,11 +47,121 @@ export const EventLogTab: React.FC = () => {
     },
   });
 
+  type logEntry = {
+    timestamp: Date;
+    user: string;
+    before?: RepoEventSettingsFragment | ProjectEventSettingsFragment;
+    after?: RepoEventSettingsFragment | ProjectEventSettingsFragment;
+  };
+
+  const eventData = isRepo
+    ? repoEventData?.repoEvents?.eventLogEntries
+    : projectEventData?.projectEvents?.eventLogEntries;
+
+  return (
+    <div data-cy="event-log">
+      {((eventData as logEntry[]) || []).map(
+        ({ user, timestamp, before, after }) => (
+          /* @ts-expect-error */
+          <EventLogCard key={`event_log_${timestamp}`}>
+            <EventLogHeader user={user} timestamp={timestamp} />
+            <Table
+              data={getEventDiffLines(before, after)}
+              columns={[
+                <TableHeader
+                  key="key"
+                  label={<span data-cy="key">Property</span>}
+                  sortBy={(datum: EventDiffLine) => datum.key}
+                />,
+                <TableHeader
+                  key="before"
+                  label={<span data-cy="before">Before</span>}
+                  sortBy={(datum: EventDiffLine) => datum.before}
+                />,
+                <TableHeader
+                  key="after"
+                  label={<span data-cy="after">After</span>}
+                  sortBy={(datum: EventDiffLine) => datum.after}
+                />,
+              ]}
+            >
+              {({ datum }) => (
+                <Row key={datum.key} data-cy="event-log-table-row">
+                  <Cell>
+                    <StyledCell>{datum.key ?? ""}</StyledCell>
+                  </Cell>
+                  <Cell>
+                    <StyledCell>{getEventValue(datum.before)}</StyledCell>
+                  </Cell>
+                  <Cell>
+                    <StyledCell>{getEventValue(datum.after)}</StyledCell>
+                  </Cell>
+                </Row>
+              )}
+            </Table>
+          </EventLogCard>
+        )
+      )}
+    </div>
+  );
+};
+
+interface Props {
+  timestamp: Date;
+  user: string;
+}
+
+const EventLogHeader: React.VFC<Props> = ({ user, timestamp }) => (
+  <StyledHeader>
+    <H3>{getDateCopy(timestamp)}</H3>
+
+    <div> {user} </div>
+  </StyledHeader>
+);
+
+/* @ts-expect-error */
+const EventLogCard = styled(Card)`
+  margin-bottom: 48px;
+  padding: ${size.m};
+  width: 150%;
+`;
+
+const StyledCell = styled("div")`
+  word-break: break-all;
+  font-size: 12px;
+  font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
+`;
+
+const StyledHeader = styled("div")`
```

Comment:
[nit] Would something like `H2`, `H3` or `Subtitle` from the LG typography package, which may have padding built in, work here? If not, applying these styles to a header element like `h3` is more semantically correct than a `div`, which is a block element.

Reply:
I'm already using H3, I need the padding on the entire header which includes the user field as well